modules/caddyhttp/tracing/module.go

```diff
@@ -25,7 +25,8 @@ func init() {
 type Tracing struct {
 	// SpanName is a span name. It should follow the naming guidelines here:
 	// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#span
-	SpanName string `json:"span"`
+	SpanName                 string `json:"span"`
+	InjectServerTimingHeader bool   `json:"injectServerTimingHeader"`
```

Comment:
If `false` is the assumed default, `omitempty` cleans up the adapted JSON. 

```suggestion
	InjectServerTimingHeader bool   `json:"injectServerTimingHeader,omitempty"`
```
We also prefer snake-case. Is this a common label? Or can it be shorter?

Disclaimer: I'm not familiar with otel

Reply:
corrected, thank you!